src/index.spec.ts

```diff
@@ -31,7 +31,11 @@ describe('Mnoga', () => {
 
     callback();
 
-    expect(handler.calledOnce).to.equal(expected);
```

Comment:
`callsSubscriber(handler, false)` would pass if the handler gets called multiple times, because this uses `calledOnce`. Of course, it would be really weird if any method of `Mnoga` ever called the handler more than once, but I wanted to make the test more strict just in case.